### DISCUSSION TITLE
Proposal to change how we manage backports in pull requests

### DISCUSSION THREAD

**Discussion by mnencia** [cloudnative-pg/cloudnative-pg#5103] [Maintainers room]
I believe our backport process needs revision. The current default of adding a "backport-requested" label results in backporting too many changes. Conversely, a "do-not-backport" default could lead to missing critical fixes in older release branches. The underlying issue seems to be the existence of a default label at all. I propose the following changes:

1. Stop adding the "backport-requested" label by default.
2. Require that every PR have either a "backport-requested" or "do-not-backport" label before merging.

This approach would ensure we always consider the necessity of backporting a change before clicking the "Squash and Merge" button.

**Comment by leonardoce** [comment]
I agree with you. We were misled by the default labels too many times.

**Comment by phisco** [comment]
Makes sense to me 👍
